threedworld-mit/tdw
# Added documentation for Resonance Audio exceptions on OS X

Closes #200 

How to test:

1. In the `tdw` repo, `git fetch` and `git checkout resonance_audio_osx`
2. Run the `audio.py` example controller

Result: `audio.py` works as expected.

See osx.md documentation for how to circumvent Resonance Audio exceptions
